net-mgmt/snmp/pkg-descr

```diff
@@ -0,0 +1,6 @@
+ bsnmp. It allows you to use SNMPv1 and SNMPv2 to query your OPNsense appliance
```

Comment:
https://github.com/opnsense/ports/blob/master/net-mgmt/bsnmp-regex/pkg-descr
https://github.com/opnsense/ports/blob/master/net-mgmt/bsnmp-ucd/pkg-descr

We would have to rewrite the descriptions but the main content could be used.